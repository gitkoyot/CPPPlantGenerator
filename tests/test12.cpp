
#include "test12.h"

int FirstClass::firstField = 0;

char*
FirstClass::firstMethod()
{
}
