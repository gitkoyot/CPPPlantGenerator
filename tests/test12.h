#ifndef TEST_1_H
#define TEST_1_H

class FirstClass
{
	static int firstField;
	int fieldThatIsNonStatic;
    static char* firstMethod();
};





#endif
