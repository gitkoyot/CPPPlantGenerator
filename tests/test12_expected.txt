@startuml
class FirstClass{
{static}-firstField   :   int
-fieldThatIsNonStatic   :   int
-{static}char * firstMethod ( )
}
@enduml

